spec/rails_helper.rb

```diff
@@ -29,7 +29,9 @@
 # run twice. It is recommended that you do not name files matching this glob to
 # end with _spec.rb. You can configure this pattern with the --pattern
 # option on the command line or in ~/.rspec, .rspec or `.rspec-local`.
-Dir[Rails.root.join('spec', 'support', '**', '*.rb')].sort.each { |f| require f }
+Dir[Rails.root.join('spec', 'support', '**', '*.rb')].sort.each do |f|
+  require f unless f.end_with?('_spec.rb')
```

Comment:
Per the above warning, putting a `_spec.rb` file in here would make it run on _every_ rspec run. It seems like it should be ok to just not automatically require any `_spec.rb` files.

Reply:
should we update the comment then?